lib/android/src/androidTest/java/net/skyscanner/backpack/reactnative/rating/BpkRatingViewManagerTest.kt

```diff
@@ -0,0 +1,36 @@
+package net.skyscanner.backpack.reactnative.rating
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.facebook.react.bridge.JavaOnlyArray
+import com.facebook.react.bridge.JavaOnlyMap
+import com.facebook.react.uimanager.ReactStylesDiffMap
+import net.skyscanner.backpack.rating.BpkRating
+import net.skyscanner.backpack.reactnative.testing.ReactViewManagerTestRule
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkRatingViewManagerTest {
+
+  @get:Rule
+  val managerRule = ReactViewManagerTestRule(BpkRatingViewManager::class)
+  private val manager: BpkRatingViewManager
+    get() = managerRule.manager
+
+  @Test
+  fun test_title() {
+    val view =  manager.createViewInstance(managerRule.themedContext)
+    manager.updateProperties(view, buildProps("title", JavaOnlyArray.of("low", "med", "high")))
```

Comment:
This is inspired by what the RN codebase does, what is basically happening here is we are testing the process of setting the `title` prop from the JS code.

We could have called `setTitle` directly in `BpkRatingViewManager` but that would not guarantee a prop with this name is available in the native side (e.g. if we forget to annotate the method with the prop name)

These are the most important "unit" tests as most of our logic for bridges is around transforming and setting props coming from the JS code. 

The next most important test are e2e, a.k.a visual regression, a.k.a screenshot test. But those will require quite a lot more work to setup.